substrate/frame/identity/src/benchmarking.rs

```diff
@@ -626,45 +623,16 @@ mod benchmarks {
 			.saturating_add(T::BasicDeposit::get());
 		let expected_sub_deposit = T::SubAccountDeposit::get(); // only 1
 
-		#[extrinsic_call]
-		_(RawOrigin::Signed(caller), target_lookup);
-
-		assert_last_event::<T>(
-			Event::<T>::DepositUpdated {
-				who: target,
-				identity: expected_id_deposit,
-				subs: expected_sub_deposit,
-			}
-			.into(),
-		);
-
-		Ok(())
-	}
-
-	#[benchmark]
-	fn lock_pallet() -> Result<(), BenchmarkError> {
-		let origin =
-			T::LockerOrigin::try_successful_origin().map_err(|_| BenchmarkError::Weightless)?;
-		Identity::<T>::unlock_pallet(origin.clone())?;
-
-		#[extrinsic_call]
-		_(origin as T::RuntimeOrigin);
-
-		assert!(Locked::<T>::get());
-
-		Ok(())
-	}
-
-	#[benchmark]
-	fn unlock_pallet() -> Result<(), BenchmarkError> {
-		let origin =
-			T::LockerOrigin::try_successful_origin().map_err(|_| BenchmarkError::Weightless)?;
-		Identity::<T>::lock_pallet(origin.clone())?;
+		#[block]
+		{
+			let _ = Identity::<T>::poke_deposit(&target);
```

Comment:
may be worth removing new benchs funcs from the pallet. new bench functions introduce new function to the weight trait, which will require all clients to run benchmarks again

Reply:
I will address this in the parent PR. Made this one more for indication of this as a better direction.